packages/google_maps_flutter/example/integration_test/src/maps_inspector.dart

```diff
@@ -533,4 +533,106 @@ void runTests() {
       expect(myLocationButtonEnabled, true);
     });
   }, skip: !isIOS);
+
+  testWidgets('marker clustering', (WidgetTester tester) async {
```

Comment:
It seems to work properly as the ClusterManager class of the platform interface is added. 
Just question: Do we need to implement more of this feature?

Reply:
https://github.com/flutter-tizen/plugins/blob/777966de1d6585143b6ec573ac41828fd1218764/packages/google_maps_flutter/example/integration_test/google_maps_test.dart#L20

maps_inspector.dart is not used test. This change was added only for code sync.

> Just question: Do we need to implement more of this feature?

Yes of course, if it can works on the lwe engine, it should be implemented.

Reply:
It seemed to work on LWE, am I wrong?